### DISCUSSION TITLE
Blender vs Bforartist turorials.

### DISCUSSION THREAD

**Discussion by jinujohnm** [Bforartists/Bforartists#2571] [Ideas]
How easy or tough is it to follow a regular blender tutorial using a bforartist?

**Comment by ReinerBforartists** [comment]
Hi jinujohnm,

When you know the tools and Bforartists a bit then it is easy. Since the tools are the very same. I have no problem to follow Blender tutorials with Bforartists at all.

But it depends of course of the tutorial. There are some really bad Blender tutorials around which just explains what hotkey to press next instead of explaining the needed steps. And our keymap is a bit different here and there. In this case you could simply switch to the Blender keymap. Or even use Blender to understand what is going on and then return.

Kind regards
Reiner

**Comment by jinujohnm** [comment]
Hi Reiner,
You are right, there are some really good tutorials which tells the whys and the hows. Thanks a lot for the answer.
